### PR TITLE
docs(cookbook): file upload recipe

### PR DESCRIPTION
**Pull request by maciejkorsan**

### CODE REVIEW

File: docs/cookbook/web/basics/file-upload.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# File upload using data objects
+
+* Preparation: **5 minutes**
+* Requirements:
+  * Initiated Syncano project
+
+### Problem to solve
+
+You want to upload file to the server
```
Comment (mkucharz) [inline docs/cookbook/web/basics/file-upload.md]: Missing dot. Maybe we should call it more precise "you want to upload file to database"?

File: docs/cookbook/web/basics/file-upload.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+### Solution
+
+You'll have to create form with upload field and socket to handle it.
+
+
+#### Prepare socket
+
```
Comment (mkucharz) [inline docs/cookbook/web/basics/file-upload.md]: Maybe we should have it more direct like:
- create socket `s create test123`
- edit `socket.yml`
- add form-data dependency `npm install --save ...`

I'm a little bit concerned that newbies might be lost somewhere on the way...

File: docs/cookbook/web/basics/file-upload.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    response.json({
+      message: 'Something went wrong'
+    }, 400)
+  }
+}
+
+
+```
+
+
```
Comment (mkucharz) [inline docs/cookbook/web/basics/file-upload.md]: 3 lines of break?

File: docs/cookbook/web/basics/file-upload.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+#### Create client side code
```
Comment (mkucharz) [inline docs/cookbook/web/basics/file-upload.md]: "client-side"

File: docs/cookbook/web/basics/file-upload.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+Let's assume that you have a simple form in html
```
Comment (mkucharz) [inline docs/cookbook/web/basics/file-upload.md]: "HTML"

File: docs/cookbook/web/basics/file-upload.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+    formData.append("filename", file.name);
+    console.log(formData)
+    s.post("application/upload", formData).then(res => {
+      console.log(res);
+    });
+  });
+
+```
+
+After submitting the file, data object will store link to your file hosted on cloudfront. File size limit is 128 MB.
```
Comment (mkucharz) [inline docs/cookbook/web/basics/file-upload.md]: File is hosted on "S3" behind the "CloudFront" - I'm only not sure if it is relevant for the end user. Upload limit to the script is 6MB. Limit for file upload from inside the Socket script is 128MB.

**Comment by maciejkorsan** [comment]
@mkucharz - updated

**Pull request merged**